app/ui.js

```diff
@@ -1053,18 +1053,17 @@ var UI;
             }
         },
 
-        connect: function() {
+        connect: function(_, password) {
```

Comment:
Oh, that's a good point. I'll change the name of the parameter. The first parameter is used for the `MouseEvent` in an `OnClick` handler.